realm/realm-library/src/main/java/io/realm/internal/LinkView.java

```diff
@@ -111,6 +111,10 @@ public void clear() {
         nativeClear(nativePointer);
     }
 
+    public boolean contains(long tableRowIndex) {
+        return nativeFind(nativePointer, tableRowIndex);
```

Comment:
I would change the nativeFind method so it returns the index. Then we can also use it to improve the performance of `indexOf`. This check should then be changed to "NOT_FOUND != nativeFind()"